gallery_dl/extractor/furaffinity.py

```diff
@@ -132,11 +135,16 @@ def _parse_post(self, post_id):
             data["_description"] = extr(
                 '<td valign="top" align="left" width="70%" class="alt1" '
                 'style="padding:8px">', '                               </td>')
+            data["folders"] = []  # folders not present in old layout
 
         data["artist_url"] = data["artist"].replace("_", "").lower()
         data["user"] = self.user or data["artist_url"]
         data["date"] = text.parse_timestamp(data["filename"].partition(".")[0])
         data["description"] = self._process_description(data["_description"])
+        data["thumbnail"] = (
+            'https://t.furaffinity.net/' +
+            str(data['id']) + '@600-' +
+            data['url'].split('/')[-2] + '.jpg')
```

Comment:
```suggestion
        data["thumbnail"] = "https://t.furaffinity.net/{}@600-{}.jpg".format(
            post_id, data["url"].rsplit("/", 2)[1])
```
A format string is simpler and `rsplit` is a bit faster here.